### DISCUSSION TITLE
USBXI support to combine more than one 6022 ?

### DISCUSSION THREAD

**Discussion by pidou46** [OpenHantek/OpenHantek6022#199] [Ideas]
Kudos to you guys to bring this amazing piece of software to us!
It cover all my needs and more.
My feeling you are done with it.
Maybe supporting USBXI interface would be a thing to bring 4 (or more) channel with a second scope?

**Comment by Ho-Ro** [comment]
There is no chance to cascade more than one 6022. This mystical USBXI isn't useful. It provides - in best case - an USB connection with strange connector; on 6022BL I was told that it also exhibits two port pins of the Cypress EsUSB main processor on 6022BL. 

![image](https://user-images.githubusercontent.com/12542359/123552720-a38dee80-d777-11eb-8f8f-2c9c2063d9df.png)
![image](https://user-images.githubusercontent.com/12542359/123552728-b274a100-d777-11eb-9c48-0b4eab2c01dd.png)

**Reply by pidou46** [reply]
Does theses pins would be clock signal ?
This would allow to synchronize devices, but each device would need is own USB port to upload data to PC.

**Reply by Ho-Ro** [reply]
No, it's only USB D+ / D- connection.
